apps/design/frontend/src/system_settings_screen.tsx

```diff
@@ -0,0 +1,559 @@
+                        : marginal,
+                    },
+                  });
+                }}
+                step={0.01}
+                min={0}
+                max={1}
+                disabled={!isEditing}
+                required
+              />
+            </InputGroup>
+            {isScoringUnmarkedWriteIns && (
+              <InputGroup label="Write-In Area Threshold">
+                <input
+                  type="number"
+                  value={systemSettings.markThresholds.writeInTextArea}
+                  onChange={(e) => {
+                    const writeInTextArea = e.target.valueAsNumber;
+                    setSystemSettings({
+                      ...systemSettings,
+                      markThresholds: {
+                        ...(systemSettings.markThresholds || {
+                          definite: 0,
+                          marginal: 0,
+                        }),
+                        writeInTextArea: Number.isNaN(writeInTextArea)
+                          ? undefined
+                          : writeInTextArea,
+                      },
+                    });
+                  }}
+                  step={0.01}
+                  min={0}
+                  max={1}
+                  disabled={!isEditing}
+                  required
+                />
+              </InputGroup>
+            )}
+          </Column>
+        </Card>
+        {ballotTemplateId !== 'NhBallotV3' &&
+          ballotTemplateId !== 'NhBallotV3Compact' && (
+            <React.Fragment>
+              <Card>
+                <H2>Auth</H2>
+                <Column style={{ gap: '1.5rem' }}>
+                  <CheckboxButton
+                    label="Enable Poll Worker PINs"
+                    isChecked={Boolean(
+                      systemSettings.auth.arePollWorkerCardPinsEnabled
+                    )}
+                    onChange={(isChecked) =>
+                      setSystemSettings({
+                        ...systemSettings,
+                        auth: {
+                          ...systemSettings.auth,
+                          arePollWorkerCardPinsEnabled: isChecked,
+                        },
+                      })
+                    }
+                    disabled={!isEditing}
+                  />
+                  <InputGroup label="Inactive Session Time Limit">
+                    <SearchSelect
+                      ariaLabel="Inactive Session Time Limit"
+                      isMulti={false}
+                      isSearchable={false}
+                      value={systemSettings.auth.inactiveSessionTimeLimitMinutes.toString()}
+                      options={[
+                        { value: '10', label: '10 minutes' },
+                        { value: '15', label: '15 minutes' },
+                        { value: '20', label: '20 minutes' },
+                        { value: '30', label: '30 minutes' },
+                        { value: '360', label: '6 hours' },
```

Comment:
Would it simplify things a bit to use number `value`s for these dropdowns? I think that's supported with `SearchSelect` (unless there are complications with going that route)

Reply:
`SearchSelect` option values must extend `string`: https://github.com/votingworks/vxsuite/blob/ac70a182ffbda2960c6e0e8328389a4d97fbfbb9/libs/ui/src/search_select.tsx#L60-L63

In retrospect, it might have been worth it to relax this type and build number support into `SearchSelect`, but probably out of scope for this change.